Libraries/Components/View/ViewPropTypes.windows.js

```diff
@@ -0,0 +1,431 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+'use strict';
+
+const React = require('React');
+const EdgeInsetsPropType = require('EdgeInsetsPropType');
+const PlatformViewPropTypes = require('PlatformViewPropTypes');
+const PropTypes = require('prop-types');
+const StyleSheetPropType = require('StyleSheetPropType');
+const ViewStylePropTypes = require('ViewStylePropTypes');
+
+const {
+  AccessibilityComponentTypes,
+  AccessibilityTraits,
+} = require('ViewAccessibility');
+
+import type {
+  AccessibilityComponentType,
+  AccessibilityTrait,
+} from 'ViewAccessibility';
+import type {EdgeInsetsProp} from 'EdgeInsetsPropType';
+import type {TVViewProps} from 'TVViewPropTypes';
+import type {Layout, LayoutEvent} from 'CoreEventTypes';
+
+const stylePropType = StyleSheetPropType(ViewStylePropTypes);
+
+export type ViewLayout = Layout;
+export type ViewLayoutEvent = LayoutEvent;
+
+// There's no easy way to create a different type if (Platform.isTVOS):
+// so we must include TVViewProps
+export type ViewProps = {
+  accessible?: bool,
+  accessibilityLabel?: null | React$PropType$Primitive<any> | string | Array<any> | any,
+  accessibilityActions?: Array<string>,
+  accessibilityComponentType?: AccessibilityComponentType,
+  accessibilityLiveRegion?: 'none' | 'polite' | 'assertive',
+  importantForAccessibility?: 'auto'| 'yes'| 'no'| 'no-hide-descendants' | 'yes-dont-hide-descendants',
+  accessibilityTraits?: AccessibilityTrait | Array<AccessibilityTrait>,
```

Comment:
It's unfortunate that we need to override this whole file just for this. Are there no other options?

Reply:
If we asked React Native to move `...PlatformViewPropTypes` to the end of this file, would that allow us to override?

---
In reply to: [184756620](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184756620) [](ancestors = 184756620)

Reply:
Yeah, that was my proposal too, maybe we're lucky and "last one wins"

---
In reply to: [184756747](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184756747) [](ancestors = 184756747,184756620)

Reply:
Yes, I think it would! Do you want me to try it with Facebook?

---
In reply to: [184756747](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184756747) [](ancestors = 184756747,184756620)

Reply:
@zholobov please do file a PR upstream, I can also do this.